wdl-format/src/format/metadata.rs

```diff
@@ -0,0 +1,324 @@
+//! A module for formatting metadata sections (meta and parameter_meta).
+
+use anyhow::Result;
+use wdl_ast::v1::LiteralNull;
+use wdl_ast::v1::MetadataArray;
+use wdl_ast::v1::MetadataObject;
+use wdl_ast::v1::MetadataObjectItem;
+use wdl_ast::v1::MetadataSection;
+use wdl_ast::v1::MetadataValue;
+use wdl_ast::v1::ParameterMetadataSection;
+use wdl_ast::AstNode;
+use wdl_ast::SyntaxElement;
+use wdl_ast::SyntaxKind;
+
+use super::comments::format_inline_comment;
+use super::comments::format_preceding_comments;
+use super::format_state::SPACE;
+use super::FormatState;
+use super::Formattable;
+use super::NEWLINE;
+
+impl Formattable for LiteralNull {
+    fn format(&self, buffer: &mut String, _state: &mut FormatState) -> Result<()> {
+        buffer.push_str("null");
+        Ok(())
+    }
+
+    fn syntax_element(&self) -> SyntaxElement {
+        SyntaxElement::Node(self.syntax().clone())
+    }
+}
+
+impl Formattable for MetadataObject {
+    fn format(&self, buffer: &mut String, state: &mut FormatState) -> Result<()> {
+        format_preceding_comments(&self.syntax_element(), buffer, state, false)?;
+
+        let open_brace = self
+            .syntax()
+            .children_with_tokens()
+            .find(|element| element.kind() == SyntaxKind::OpenBrace)
+            .expect("Metadata Object should have an open brace");
+        format_preceding_comments(&open_brace, buffer, state, true)?;
+        // Open braces should ignore the "+1 rule" followed by other interrupted
+        // elements.
+        if state.interrupted() {
+            state.reset_interrupted();
+            state.indent(buffer)?;
+        }
+        buffer.push('{');
+        format_inline_comment(&open_brace, buffer, state, false)?;
+
+        state.increment_indent();
+
+        let mut commas = self
+            .syntax()
+            .children_with_tokens()
+            .filter(|c| c.kind() == SyntaxKind::Comma);
+        for item in self.items() {
```

Comment:
```suggestion

        for item in self.items() {
```
When I initially read this, it ran together with `commas`.